src/Xamarin.Android.Build.Tasks/Tasks/GenerateJavaStubs.cs

```diff
@@ -251,8 +251,8 @@ void Run (DirectoryAssemblyResolver res)
 				template => template.Replace ("// REGISTER_APPLICATION_AND_INSTRUMENTATION_CLASSES_HERE", regCallsWriter.ToString ()));
 			
 			// Create NotifyTimeZoneChanges java sources.
-			string notifyTimeZoneChangesFile = "NotifyTimeZoneChanges.java";
-			SaveResource (notifyTimeZoneChangesFile, notifyTimeZoneChangesFile, real_app_dir, template => template);
+			//string notifyTimeZoneChangesFile = "NotifyTimeZoneChanges.java";
+			//SaveResource (notifyTimeZoneChangesFile, notifyTimeZoneChangesFile, real_app_dir, template => template);
```

Comment:
Commented code, probably meant to be removed?

If so, it looks like `SaveResource` is only used in one place now. Should we just remove the `SaveResource` method and put the code in-line here?